src/vorta/views/archive_tab.py

```diff
@@ -64,11 +68,13 @@ def __init__(self, parent=None, app=None):
         super().__init__(parent)
         self.setupUi(parent)
         self.mount_points = {}  # mapping of archive name to mount point
+        self.quick_mount_points = {}  # mapping of archive name to quick mount point
```

Comment:
Maybe the `self.mount_points` dict can be reused for this. Why a new dict for this? Not possible to add more than one archive from the same repo and the mounts need to be discovered during startup. So tracking them all in one place is better.